plugins/register-component/package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@backstage/catalog-model": "^0.1.1-alpha.20",
     "@backstage/core": "^0.1.1-alpha.20",
+    "@backstage/components": "^0.1.1-alpha.20",
```

Comment:
Let's not do this imo, and just stick to depending on core

Reply:
For all the plugins from BOSS?

Reply:
For all plugins everywhere :grin:. The components would be separated out into a `@backstage/core-components` package, which would be installed, but not depended on directly. All components would be re-exported by `@backstage/core`